setup_robotino/InstallSquirrel.sh

```diff
@@ -11,17 +11,96 @@ EOF
 green='\e[0;32m'
 red='\e[0;31m'
 NC='\e[0m' # No Color
+DISTRIB_CODENAME='trusty'
 
 
+#### FUNCTION check if sudo can be executed 
+function CheckUserSudo () {
+  echo -e "\n${green}CheckUserSudo got $1 as argument${NC}\n"
+  #if [[ $USER != $1 ]]
+  if [[ `whoami` -ne $1 ]]
+    then
+      echo -e "\n${red}FAILURE: Not running as user '$1'${NC}\n"
+      exit 1
+  fi
+  echo -e "\n${green}INFO: Check if user $1 is able to run commands with sudo${NC}\n"
+  sleep 5 
+  sudo -v
+  if [[ $? != 0 ]]
+    then 
+      echo -e "\n${red}FAILURE: $1 is not able to run sudo ${NC}\n"
+      exit 1
+  fi
+}
+
+#### FUNCTION CREATE squirrel USER
+function CreateSquirrelUser {
+  ret=false
+  getent passwd squirrel >/dev/null 2>&1 && ret=true
+  
+  if $ret; then
+    echo -e "\n${green}INFO: User squirrel already exists${NC}\n"
+  else
+    echo -e "\n${green}INFO: Create user 'squirrel'${NC}\n"
+    sudo adduser squirrel
+    if [[ $? != 0 ]]
+      then 
+        echo -e "\n${red}FAILURE: Unable to create user 'squirrel'${NC}\n"
+        exit 1
+    fi
+  fi
+
+  for i in 'sudo' 'dialout' 'audio';
+    do
+      if groups squirrel | grep &>/dev/null $i; then
+        echo -e "\n${green}INFO: User squirrel already in $i${NC}\n"
+      else
+        echo -e "\n${green}INFO: Add user 'squirrel' to group $i${NC}\n"
+        sudo usermod -aG sudo squirrel
+        if [[ $? != 0 ]]
+          then 
+            echo -e "\n${red}FAILURE: Unable to add squirrel to $i${NC}\n"
+            exit 1
+        fi
+      fi
+    done
+  
+  if sudo grep "squirrel ALL=(ALL) NOPASSWD: ALL" /etc/sudoers &> /dev/null
+    then
+      echo -e "\n${green}INFO: User squirrel is already in sudoers file${NC}\n"
+  else
+    echo -e "\n${green}INFO: Allow squirrel user to execute sudo command without password${NC}\n"
+    echo "squirrel ALL=(ALL) NOPASSWD: ALL" | sudo tee -a /etc/sudoers
+  fi    
+  if sudo grep "%sudo   ALL=(ALL:ALL) ALL" /etc/sudoers &> /dev/null
+    then
+      echo -e "\n${green}INFO: Group sudo is already in sudoers file${NC}\n"
+  else
+    echo -e "\n${green}INFO: Allow group sudo to execute sudo${NC}\n"
+    echo "%sudo   ALL=(ALL:ALL) ALL" | sudo tee -a /etc/sudoers
+  fi    
+}
+
+#### FUNCTION ROS INSTALLATION
+function RosInstallation {
+  echo -e "\n${green}INFO: Setup ROS${NC}\n"
+  sudo sh -c '. /etc/lsb-release && echo "deb http://packages.ros.org.ros.informatik.uni-freiburg.de/ros/ubuntu $DISTRIB_CODENAME main" > /etc/apt/sources.list.d/ros-latest.list'
+  sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 0xB01FA116
+  sudo apt-get update
+  sudo apt-get install ros-indigo-ros-base
```

Comment:
why base and not desktop version? you will also need `ros-indigo-rosbash and python-ros-*`

add the arguments  `-y --force-yes` to apt-get install